src/Tests.IntegrationTests/Fixtures/CentOpsFixture.cs

```diff
@@ -21,75 +22,88 @@ public CentOpsFixture(IConfiguration configuration, TestClients testClients)
+                    Status = "Active",
+                });
+                var institution = _testClients.CentOpsAdminClient.Request<Institution>(Verb.Post, _institutionsUri, institutionPostBody).Result;
 
-            // Create Classifier
-            var classifierPostBody = JsonSerializer.Serialize(new Participant()
-            {
-                Name = "classifier1",
-                InstitutionId = institution.Id,
-                Host = "http://classifier/dmr-api/messages",
-                Type = "Classifier",
-                Status = "Active",
-                ApiKey = "thisisareallylongkeyforclassifier"
-            });
-            _ = _testClients.CentOpsAdminClient.Request<Participant>(Verb.Post, _participantsUri, classifierPostBody).Result;
-
-            // Create Dmr
-            var dmrPostBody = JsonSerializer.Serialize(new Participant()
-            {
-                Name = "dmr1",
-                InstitutionId = institution.Id,
-                Host = "http://dmr/messages",
-                Type = "Dmr",
-                Status = "Active",
-                ApiKey = "thisisareallylongkey"
-            });
-            _ = _testClients.CentOpsAdminClient.Request<Participant>(Verb.Post, _participantsUri, dmrPostBody).Result;
-
-            // Create Bot1
-            var bot1PostBody = JsonSerializer.Serialize(new Participant()
-            {
-                Name = "bot1",
-                InstitutionId = institution.Id,
-                Host = "http://bot1/dmr-api/messages",
-                Type = "Chatbot",
-                Status = "Active",
-                ApiKey = "thisisareallylongkeyformockbot1"
-            });
-            _ = _testClients.CentOpsAdminClient.Request<Participant>(Verb.Post, _participantsUri, bot1PostBody).Result;
+
+                // Create Classifier
+                var classifierPostBody = JsonSerializer.Serialize(new Participant()
+                {
+                    Name = $"classifier1",
+                    InstitutionId = institution.Id,
+                    Host = $"{_configuration["ClassifierInternalUrl"]}/dmr-api/messages",
+                    Type = "Classifier",
+                    Status = "Active",
+                    ApiKey = "thisisareallylongkeyforclassifier"
+                });
+                _ = _testClients.CentOpsAdminClient.Request<Participant>(Verb.Post, _participantsUri, classifierPostBody).Result;
+
+                // Create Dmr
+                var dmrPostBody = JsonSerializer.Serialize(new Participant()
+                {
+                    Name = $"dmr1",
+                    InstitutionId = institution.Id,
+                    Host = $"{_configuration["DmrInternalUrl"]}/messages",
+                    Type = "Dmr",
+                    Status = "Active",
+                    ApiKey = "thisisareallylongkey"
+                });
+                _ = _testClients.CentOpsAdminClient.Request<Participant>(Verb.Post, _participantsUri, dmrPostBody).Result;
```

Comment:
HttpClient.PostAsJsonAsync might be a bit more concise if it fits in here.

Reply:
The `.Request` extension method does a few other bits like ensure success and deserialise the response, so would prefer to keep that consistently used and leave it like it is, hope that is ok.